src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -936,5 +942,11 @@ internal override void UpdateThemeBindings()
 		}
 
 		internal override bool CanHaveChildren() => true;
+
+		internal override bool IsFocusable =>
+			/*IsActive() &&*/ //TODO Uno: No concept of IsActive in Uno yet.
```

Comment:
Is this corresponds to the [`IsInLiveTree`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.MuxInternal.cs#L10) ?

Reply:
I am afraid it could break some `NET461` tests though (as `IsFocusable` is often queried and it currently seems to throw)